lib/elixir_make/artefact.ex

```diff
@@ -376,7 +376,7 @@ defmodule ElixirMake.Artefact do
   end
 
   defp warning_if_no_cacertfile!(nil) do
-    Logger.warning("""
+    Logger.warn("""
```

Comment:
.warn is deprecated. But perhaps it is better to use `Mix.shell().error(...)` anyway?

Reply:
Sure! Actually, shall we use `Mix.shell().{info(...)/error(...)}` instead of `Logger`?

Reply:
@cocoa-xu I wouldn't worry about it for now. One of the last things for me to do is to review the error messaging throughout the codebase. :)

Reply:
Okay! I'll push the code in a minute!